data/Santini_2016/metadata.yml

```diff
@@ -0,0 +1,349 @@
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: root wood density replicates
+  methods: Root material collected for assessment of wood density was stored in paper
+    bags and kept cool for laboratory analysis. Wood density of roots was measured
+    in 4-cm lengths of sapwood. Sample numbers of individual roots (i.e. one root
+    sample per plant) for assessment of wood density were as follows, Erythrina vespertilio
+    (n=5), Acacia aneura (n=5), Hakea sp. (n=5), Psydrax latifolia (n=5), Corymbia
+    opaca (n=3) and Eucalyptus camaldulensis (n=5). Root sapwood segments were weighed
+    after drying to a constant mass. Wood density was calculated as the ratio of dry
+    weight (5 days at 60°C) to volume of water displaced by the fully hydrated sapwood
+    segments (i.e., after they were submerged in water for two days; see Santini et
+    al. 2016).
+- var_in: stem Sapwood density (g cm-3)
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 6
+  method_context: sapwood
+  methods: Material to measure density of intact branches and wood density was stored
+    in paper bags and was maintained cool in an insulated container until laboratory
+    analysis. Density of intact branches was measured from ~5 cm length branch segments.
+    Branches were placed in water for 2 days to obtain full branch hydration. We measured
+    the mass of the displaced hydrated branch. The same branches were oven dried for
+    5 days at 60 °C to attain constant weight. Density of intact branches was calculated
+    by dividing dry mass by hydrated branch volume. Wood density was measured following
+    the same methods as for measuring density of intact branches but after carefully
+    debarking our intact branches with a razor blade.
+- var_in: wood density total, including bark and pith (g/cm3)
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 6
+  method_context: total wood
+  methods: Material to measure density of intact branches and wood density was stored
+    in paper bags and was maintained cool in an insulated container until laboratory
+    analysis. Density of intact branches was measured from ~5 cm length branch segments.
+    Branches were placed in water for 2 days to obtain full branch hydration. We measured
+    the mass of the displaced hydrated branch. The same branches were oven dried for
+    5 days at 60 °C to attain constant weight. Density of intact branches was calculated
+    by dividing dry mass by hydrated branch volume. Wood density was measured following
+    the same methods as for measuring density of intact branches but after carefully
+- var_in: stem Vessel wall thickness (um)
+  unit_in: um
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
```

Comment:
Should replicates be 20-30 because of "We also assessed vessel wall thickness by measuring double-vessel walls of 20–30 vessels per species and dividing the total distance by two."?

Reply:
Except we only have 1 value per taxon, so it is the number of individuals in this case. 9 still wrong - and has been updated